pkg/serverless/metrics/metric.go

```diff
@@ -124,3 +129,9 @@ func buildBufferedAggregator(multipleEndpointConfig MultipleEndpointConfig, forw
 func buildMetricBlocklist(userProvidedList []string) []string {
 	return append(userProvidedList, invocationsMetric)
 }
+
+// Need to account for duplicate metrics when using the proxy.
+// Can be removed when the proxy is removed
+func buildMetricBlocklistForProxy(userProvidedList []string) []string {
+	return append(userProvidedList, invocationsMetric, errorsMetric)
```

Comment:
I would maybe reuse `buildMetricBlocklist` this will avoid us to maintain two lists. Here we just want to append errorsMetric to any precomputed list